compiler/gentree.cpp

```diff
@@ -539,7 +539,17 @@ VertexPtr GenTree::get_expr_top(bool was_arrow) {
         break;
       }
       cur--;
-      res = get_func_call<op_func_call, op_none>();
+
+      // we don't support namespaces for functions yet, but we
+      // permit \func() syntax and interpret it as func();
+      // this logic is compatible with what we'll get after the
+      // function namespaces will be implemented
+      auto func_call = get_func_call<op_func_call, op_none>();
+      if (func_call->str_val[0] == '\\') {
+        func_call->str_val = func_call->str_val.substr(1);
```

Comment:
`func_calll->str_val.erase(0, 1)` won't allocate extra string

Reply:
I was using erase (with iterator), but the code that removes the class leading `\` did `substr(1)`, so I followed that pattern. :sweat_smile:

Reply:
I think than for classes we used `string_view`

Reply:
You're right. I'll change the code to use erase.